### DISCUSSION TITLE
Help with hiding the mouse cursor

### DISCUSSION THREAD

**Discussion by nookstray** [dialogic-godot/dialogic#851] [Q&A]
Just a quick question: how do I hide the mouse cursor when I’m in dialogue? I have it hidden normally, but it reappears when choices come up. Would also like to know how I could disable mouse input entirely because even if I hide the cursor, the player could still hover over the options and select them. Thanks in advance!

I thought about simply adding this,

```
if Dialogic.has_current_dialog_node() == true:
		Input.set_mouse_mode(Input.MOUSE_MODE_HIDDEN)
```

but it didn’t seem to work.

**Comment by Jowan-Spooner** [comment]
Hmm, yeah the mouse is shown automatically, because we assumed people would want the mouse to select choices 😨. 
You can just comment out the lines that make the mouse appear (DialogNode.gd function `add_choice_button` around line 1110).

You could simply put a control element in front of the screen (on a higher layer) if you want to block mouse input.

**Reply by nookstray** [reply]
Thanks for the quick reply. It worked! I hid the mouse, but I don’t quite understand how to put a control element in front of the screen. How do I do this? And do I do it for every scene, or is there a way to disable mouse input globally?
